frontend/web/components/pages/AccountSettingsPage.js

```diff
@@ -221,6 +243,21 @@ class TheComponent extends Component {
                           checked={flagsmith.getTrait('json_inspect')}
                         />
                       </Row>
+                      <Row className='mt-4' space>
```

Comment:
I'd love to see this wrapped in a feature flag!

Reply:
The feature flag `delete_account` has been created and used here to display the Delete Account feature. It is set to false for production, and true for staging.